src/ipelib/ipedoc.cpp

```diff
@@ -217,6 +217,7 @@ Document * Document::load(const char * fname, int & reason) {
     reason = EFileOpenError;
     std::FILE * fd = Platform::fopen(fname, "rb");
     if (!fd) return nullptr;
+    Platform::changeDirectory(Platform::parentDirectory(fname));
```

Comment:
We cannot simply change the current directory inside Ipe.
There are quite a few people who have ipelets that call bash scripts or that expect the cwd to be in a specific place for some other reason.

We would at least have to set it back after the bitmaps have been loaded.

However, this also seems the wrong place.   Just reading the document using `Document::load` should definitely *not* load all the bitmaps - imagine this being called from an ipescript that just wants to extract some information.

Loading the bitmaps should only happen when we need to render them.